### DISCUSSION TITLE
Retweet and claim your IBXC Error!

### DISCUSSION THREAD

**Discussion by Zeynel26477575** [IBAX-io/go-ibax#1714] [Q&A]
When I click "Finish" for "Retweet and claim your IBXC"
Saying Error that "Please try again later"

How can I solve this problem?

thx.

**Comment by CFarketmez** [comment]
up. guys click up because of the team have to see this problem

**Comment by CFarketmez** [comment]
@ibaxio can you check out this problem sir

**Comment by Zeynel26477575** [comment]
Still continue "error"
